services/API-service/src/scripts/seed-prod.ts

```diff
@@ -33,7 +34,24 @@ export class SeedProd implements InterfaceScript {
 
       await userRepository.save(adminUser);
     } else {
-      this.logger.log('Users already exist, skip seed admin user.');
+      this.logger.log(
```

Comment:
1. This block should also update the dunant user's countries, similar to `seed-init`

```typescript
dunantUser.countries = await countryRepository.find();
```

2. Update the IF block to use `DUNANT_EMAIL` as in the ELSE block instead of `UserRole.Admin`.